src/hooks/useHandleCreateWrapped.tsx

```diff
@@ -795,6 +849,15 @@ export function useHandleCreateWrapped(
       !!signedVAA
     ) {
       sui(dispatch, enqueueSnackbar, suiWallet, signedVAA, foreignAddress);
+    } else if (isCosmosChain(targetChain as any) && !!signedVAA) {
```

Comment:
not all the cosmos chains should end into it, for instance Sei and Injective do have special cases, we could talk about it

Reply:
Done!